Makefile.eia

```diff
@@ -0,0 +1,64 @@
+.PHONY: all pull fetch patch generate clean codegen mod docs move-other patch-post fmt test stage
+
+include Makefile
+
+PACKAGE_NAME=eia
+SPEC_BASE_URL:=https://api.swaggerhub.com/apis/equinix-api/Equinix_Internet_Access_API_v2/2.0
```

Comment:
The onboard-service task recommends including the version in the service name (e.g., `metalv1`, `fabricv4`).  However it looks like in the case of the Internet Access spec, the "v2" spec covers both v1 and v2; am I reading that correctly?  If so, then I can see why it makes sense to onboard this service as `eia` rather than `eiav#`

The reason for including the version tag in the service name for the SDK is to provide an easier upgrade path when, e.g., Metal v2 pops into existence someday: at that point, `metalv2` would be onboarded as a separate service, and users would migrate from one package to the other before the v1 API was eventually shut down.  Do you know if there is a plan to eventually separate the EIA v1 and v2 specs?

Reply:
I can adjust it to be `eiav2`. Overlooked the versioning number when onboarding but it should only be for v2 and not v1.